src/npr/rules_dorado/00_prepare_bam.smk

```diff
@@ -0,0 +1,63 @@
+'''
+Prepare BAM file
+As the new ONT machine can provide basecalls using dorado + genome
+This part checks if we have the BAM files and merge
+'''
+
+rule prepare_bam:
+    input: 
+        "flags/00_prepare.done"
+    output:
+        flag=touch("flags/00_prepare_bam.done"),
+    params:
+        idir=config["info_dict"]["base_path"],
+        baseout=os.path.join(config['info_dict']['flowcell_path'], "bam"),
+        batch_size=500,
+        opt='-c --no-PG -@'
```

Comment:
Batch size and opt is something for the config ?

Reply:
yes, I will add options for this